package.json

```diff
@@ -6,9 +6,10 @@
     "npm": ">= 8.0.0 < 9.0.0"
   },
   "devDependencies": {
-    "prettier": "2.8.1",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
I think this should rather be within the app itself. That way we could easily split up the monorepo in the future.

Reply:
Thanks good catch. Didn't even realized I'd installed this at the top level 🤦‍♂️